insights/src/main/kotlin/com/algolia/instantsearch/insights/webservice/WebServiceHttp.kt

```diff
@@ -16,23 +17,17 @@ internal class WebServiceHttp(
 ) : WebService {
 
 
-    enum class Environment(private val baseUrl: String) {
+    enum class Environment(baseUrl: String) {
         Prod("https://insights.algolia.io"),
         Debug("http://localhost:8080");
 
-        fun buildUrl(eventType: EventType): String {
-            return "$baseUrl/1/searches/${eventType.route}"
-        }
+        val url: String = "$baseUrl/1/events"
     }
 
-    override fun sendEvent(event: Event): WebService.Response {
-        val eventType = when (event) {
-            is Event.Click -> EventType.Click
-            is Event.View -> EventType.View
-            is Event.Conversion -> EventType.Conversion
-        }
-        val string = ConverterParameterToString.convert(event.params)
-        val url = URL(environment.buildUrl(eventType))
+    override fun send(events: List<Event>): WebService.Response {
+        val array = JSONArray().apply { ConverterEventToString.convert(events).forEach { that: String -> put(JSONObject(that)) } }
```

Comment:
```suggestion
        val array = JSONArray(ConverterEventToString.convert(events))
```

This should be doing the same, but more concise

Reply:
Isn't the same:
```kotlin
Original: `{"events":[{"userToken":"foobarbaz","eventName":"EventA","index":"latency","eventType":"click","timestamp":1543827220072,"objectIDs":["54675051"]}]}`
WithSuggestion: `"{"events":["{\"userToken\":\"foobarbaz\",\"eventName\":\"EventA\",\"index\":\"latency\",\"eventType\":\"click\",\"timestamp\":1543827204560,\"objectIDs\":[\"54675051\"]}"]}"`
```

Reply:
`JSONArray(ConverterEventToString.convert(events).map(::JSONObject))`

Mapping with JSONObject should fix it 👍